backend/README.md

```diff
@@ -52,19 +60,35 @@ backend
 ```bash
 git clone https://github.com/lelouchB/final-space-api.git
 ```
+![clone step](https://i.imgur.com/icZznpW.png)
+
+2. Cd to the backend directory
+
+```bash
+cd final-space-api/backend
+```
+![cd step](https://i.imgur.com/4Pteq76.png)
 
-2. Go to the backend directory
 3. Install the required npm packages
 
 ```bash
 npm install
 ```
+![npm install step](https://i.imgur.com/iRUgixn.png)
 
-4. Create a **.env** folder and add the **Database URL** in it.
+4. Copy `.env.example` file and rename it to `.env`
 
+```bash
+cp .env.example .env
 ```
-DATABASE_URL=mongodb://127.0.0.1:27017/final-space-api
+![cp step](https://i.imgur.com/xbQPLDQ.png)
+
+5. (Optional) seed local database with actual data from api
```

Comment:
Just make this upper case API and we are good to go.

Thanks

Reply:
Done and I add a link to API 👍🏼